forest/daemon/src/daemon.rs

```diff
@@ -95,16 +95,16 @@ pub(super) async fn start(config: Config, detached: bool) -> Db {
     let mut ks = if config.client.encrypt_keystore {
         loop {
             print!("Enter the keystore passphrase: ");
-            std::io::stdout().flush().unwrap();
+            std::io::stdout().flush()?;
 
             let passphrase = read_password().expect("Error reading passphrase");
```

Comment:
`expect` is as evil as `unwrap`.

```suggestion
            let passphrase = read_password()?;
```

Reply:
Fixed